src/ddmd/dstruct.d

```diff
@@ -696,12 +706,16 @@ extern (C++) class StructDeclaration : AggregateDeclaration
     }
 
     /***************************************
-     * Return true if struct is POD (Plain Old Data).
+     * Determine if struct is POD (Plain Old Data).
+     *
      * This is defined as:
-     *      not nested
-     *      no postblits, destructors, or assignment operators
-     *      no 'ref' fields or fields that are themselves non-POD
+     *      1. not nested
+     *      2. no postblits, destructors, or assignment operators
+     *      3. no 'ref' fields or fields that are themselves non-POD
```

Comment:
You probably want to use `$(UL)$(LI)` or line breaks here.

![image](https://user-images.githubusercontent.com/4370550/26861595-302a4ea6-4b45-11e7-91bc-7e8fd96bdd1e.png)